client/webserver/site/src/js/markets.js

```diff
@@ -1657,12 +1682,14 @@ class ExchangeSection {
     this.rows = Doc.tmplElement(box, 'mkts')
     const rowTmpl = Doc.tmplElement(this.rows, 'mktrow')
     this.rows.removeChild(rowTmpl)
-    for (const mkt of Object.values(dex.markets)) {
-      this.marketRows.push(new MarketRow(rowTmpl, mkt))
-    }
-
-    for (const market of this.sortedMarkets()) {
-      this.rows.appendChild(market.row)
+    // if disconnected is not possible to get the markets from the server.
+    if (dex.markets) {
```

Comment:
Prefer less indentation, so

```suggestion
    if (!dex.markets) return
```